mopidy/core/history.py

```diff
@@ -69,7 +71,7 @@ def _save_state(self):
                 break
         return HistoryState(history=history_list)
 
-    def _load_state(self, state, coverage):
+    def _load_state(self, state: HistoryState, coverage: list[str]) -> None:
```

Comment:
With regards to our use of this variable, we could change it to `Iterable[str]`, as all we do is check `x in coverage`.

As for where it is used, `Core._setup()` creates a list that is passed as the `coverage` argument to all the `_load_state()` methods.

Reply:
Thinking more about, I actually want to replace most `list[...]` with `Iterable[...]` in the arguments, but leave `list[...]` in the return types, in the spirit of being lenient with input but conservative and explicit with output.

Reply:
I've updated all arguments from `list` to `Iterable` today. No complaints from the type checker, so seems we don't rely at e.g. index access at all. Return values are still `list` so that users of the API has as much information as possible about what they'll get in return.